companion/src/mdichild.cpp

```diff
@@ -56,7 +56,9 @@ MdiChild::MdiChild(MainWindow * parent):
   initModelsList();
   connect(parent, SIGNAL(FirmwareChanged()), this, SLOT(onFirmwareChanged()));
   connect(ui->modelsList, SIGNAL(doubleClicked(QModelIndex)), this, SLOT(openModelEditWindow()));
+  connect(ui->modelsList, SIGNAL(pressed(QModelIndex)), this, SLOT(onItemSelected(QModelIndex)));
   connect(ui->modelsList, SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(showModelsListContextMenu(const QPoint &)));
+  connect(ui->modelsList, SIGNAL(activated(QModelIndex)), this, SLOT(openModelEditWindow()));
```

Comment:
This could move up, just below/above the `doubleClicked` connect.  Much easier to see that they connect to same slot that way.